File: NetBeans_IDE/MmgGameApiJava/src/net/middlemind/MmgGameApiJava/MmgBase/MmgEventHandler.java
package net.middlemind.MmgGameApiJava.MmgBase;

/**
 * The base event handler class for event passing. 
 * Created by Middlemind Games 08/29/2016
 *
 * @author Victor G. Brusca
 */
public interface MmgEventHandler {
    
    /**
     * Handles an MmgEvent object.
     *
     * @param e     The event to handle.
     */
    public void MmgHandleEvent(MmgEvent e);
}

File: VisualStudio_IDE/MmgGameApiCs/MmgGameApiCs/src/net/middlemind/MmgGameApiCs/MmgBase/MmgEventHandler.cs
using System;

namespace net.middlemind.MmgGameApiCs.MmgBase
{
    /// <summary>
    /// The base event handler class for event passing. 
    /// Created by Middlemind Games 08/29/2016
    /// 
    /// @author Victor G. Brusca
    /// </summary>
    [System.Diagnostics.CodeAnalysis.SuppressMessage("Style", "IDE1006:Naming Styles", Justification = "<Pending>")]
    public interface MmgEventHandler
    {
        /// <summary>
        /// Handles an MmgEvent object.
        /// </summary>
        /// <param name="e">The event to handle.</param>
        public void MmgHandleEvent(MmgEvent e);
    }
}